src/components/Dashboard/TodoList.vue

```diff
@@ -122,9 +122,16 @@
                 <!-- NB: blocks below are mutually exclusive, and order is important -->
 
                 <!-- affiliation invitation todo task -->
+                <div
+                  v-if="isAffiliationInvitation(item)"
+                  class="todo-subtitle"
+                >
+                  <span v-if="!!item.subtitle">{{ item.subtitle }}</span>
+                </div>
+
                 <!-- new todo task -->
                 <div
-                  v-if="isStatusNew(item) || isAffiliationInvitation(item)"
+                  v-else-if="isStatusNew(item)"
```

Comment:
I split up the affiliation-invitation and new-todo-task cases. This works better with code-folding and de-couples the cases for future refactoring.